### DISCUSSION TITLE
What this error means

### DISCUSSION THREAD

**Discussion by lukar777** [Chia-Network/chia-blockchain#15000] [Support]
Is this is preventing me from winning block, my estimated time to win is one month i didn't win a block in 5 months.

2023-04-07T13:51:57.825 full_node full_node_server        : INFO     Cannot connect to host 18.170.113.87:8444 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1131)')]

**Comment by nurazhardotcom** [comment]
That error means that the full node server was unable to connect to another host (with IP 18.170.113.87) on port 8444 for two reasons:

The connection was using SSL encryption (ssl:True)
The SSL certificate verification failed for that connection.
Specifically, it saw a self-signed SSL certificate, but rejected it due to the CERTIFICATE_VERIFY_FAILED error.
By default, the full node will only establish SSL connections to peers if the certificate chains up to a trusted root CA. A self-signed certificate does not, so the connection is rejected.

To fix this, there are a few options:

Add the self-signed certificate as a trusted certificate, so the full node will accept the connection.
Tell the full node to disable certificate validation altogether for that host by using ssl:False instead of ssl:True in the connection.
Obtain a proper signed SSL certificate for that peer host.
Without seeing the full configuration and setup, it's hard to provide more specific guidance. Let me know if you have any other questions!

**Reply by lukar777** [reply]
Is this is preventing me from winning block??  my estimated time is one month before i was on RPI4 and it was wining the blocks as expected. 
 I switch to mini pc lenovo m910q spec I3 7100T 8gb DDR4 in 12.2022 before the Christmas and since then no wining block. Then this error is on my side (ful node) with self signed cert
